src/style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+.product-card {
+  width: 200px;
+  height: 406px;
+  border: 1px solid rgb(243 243 243);
+  border-radius: 5px;
+}
+
+.product-card__container {
+  margin: 32px 16px 16px;
+}
+
+.product-card__img {
+  margin: 0 3px 36px;
+}
+
+.card-img {
+  width: 100%;
+  height: 134px;
+}
+
+.product-card__title {
+  margin-bottom: 16px;
+}
+
+.title {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: rgb(6 11 53);
+}
+
+.product-card__id {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: rgb(97 96 112);
+}
+
+.product-card__rating {
+  margin-bottom: 22px;
+
+}
+
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+/* #region Stars */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star-active.svg);
+  background-size: cover;
+  margin: 2px 6px 2px 2px;
+  width: 12px;
+  height: 12px;
+}
+
+.stars .stars__star:nth-last-child(-n+1) {
+  background-image: url(./images/star.svg);
+  margin-right: 0;
+}
```

Comment:
Reuse stars block from Stars task and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.